lib/src/components/NxPagination/NxPagination.tsx

```diff
@@ -115,27 +135,31 @@ export default function NxPagination({ className, pageCount, currentPage, onChan
 
         { numPagesAboveRange > 1 &&
           // Rigth '...' - forward one page group
-          <NxButton className={getBtnClasses()} onClick={handleBtnClick(currentPageRangeEnd)}>
+          <NxButton aria-label={`show next ${PAGE_RANGE_SIZE} pages`}
```

Comment:
The ellipsis buttons don't just show the next set of buttons, they navigate to a different page.  I think they should have text similar to what the other buttons has which simply says which page they go to.

Reply:
3f1d6844